src/co/elastic/NotificationManager.groovy

```diff
@@ -367,19 +365,57 @@ def generateBuildReport(Map args = [:]) {
     return output
 }
 
-def queryFilter(timeout, flakyThreshold) {
-  return """{
-                "timeout": "${timeout}",
-                "sort" : [
-                  { "timestamp" : "desc" },
-                  { "test_score" : "desc" }
-                ],
-                "query" : {
-                  "range" : {
-                    "test_score" : {
-                      "gt" : ${flakyThreshold}
-                    }
-                  }
+def queryFilter(jobName) {
+  return """
+{
+  "aggs": {
```

Comment:
Do we need the default values such as `should`, `script_fields` ...?

Reply:
I massaged the query down a bit to just what I think should be needed.